test/server/server_fuzz_test.cc

```diff
@@ -97,6 +97,10 @@ bool validateLbSubsetConfig(const envoy::config::bootstrap::v3::Bootstrap& input
         if (subset_selector.keys().size() != 1) {
           return false;
         }
+        // Expect key to be non-empty when use_single_host_per_subset is set to true.
+        if (subset_selector.keys()[0].empty()) {
```

Comment:
nit: make a temp for subset_selector.keys().

Reply:
Thanks, fixed.